src/js/control-bar/text-track-controls/chapters-button.js

```diff
@@ -78,7 +78,9 @@ class ChaptersButton extends TextTrackButton {
     let items = this.items = [];
 
     for (let i = 0, length = tracks.length; i < length; i++) {
-      let track = tracks[i];
+
+      // We will always choose the last track as our chaptersTrack
+      let track = tracks[length - 1];
```

Comment:
I think for the last chapters track, due to the break statement....it does not always choose the last track. 
Let's say we have multiple chapters list and if one the in-between chapters kind matches ; it will choose a track from in-between of the list and break out from the loop. 
I wanted to always make sure that we are taking the last track no matter how big the list is.

Also track chosen in `createItems` would be different, but we are popping out the items in `createMenu` while always maintaining a chapters list, so that would not be an issue I think.

Well we actually do not need `createItems` if we decide to take this approach.